generators/java/generator-utils/src/main/java/com/fern/java/utils/NullableAnnotationUtils.java

```diff
@@ -0,0 +1,28 @@
+package com.fern.java.utils;
+
+import com.fern.ir.model.types.ContainerType;
+import com.fern.ir.model.types.TypeReference;
+import com.squareup.javapoet.AnnotationSpec;
+import com.squareup.javapoet.ClassName;
+
+public final class NullableAnnotationUtils {
+
+    private static final ClassName NULLABLE = ClassName.get("org.jetbrains.annotations", "Nullable");
```

Comment:
hm it looks like we already have a dependency on `org.jetbrains.annotations` for this. I'm kind of surprised we chose that (?) would have expected us to roll with the Spring one for the Spring generator, and the most popular one possible for the SDKs.

No need to address in this PR!